diff -urN libsigc++-1.1.9.org/acinclude.m4 libsigc++-1.1.9/acinclude.m4
--- libsigc++-1.1.9.org/acinclude.m4	Thu Jan  1 01:00:00 1970
+++ libsigc++-1.1.9/acinclude.m4	Sun Apr 21 03:07:39 2002
@@ -0,0 +1,8 @@
+AC_DEFUN(AC_PROG_CXX_SUN,
+[AC_CACHE_CHECK(whether we are using SUN CC compiler, ac_cv_prog_sun_cxx,
+[if AC_TRY_COMMAND(${CXX-g++} -V 2>&1) | egrep "Sun WorkShop" >/dev/null 2>&1; then
+  ac_cv_prog_sun_cxx=yes
+else
+  ac_cv_prog_sun_cxx=no
+fi])])
+
