--- libsigc++-1.0.0/configure.in	Sat Apr 15 04:11:21 2000
+++ libsigc++-1.0.0/configure.in~	Sun Apr 16 00:36:27 2000
@@ -559,7 +559,7 @@
     return 0;
   }
 ],[
- . ac_thread.sh
+ . ./ac_thread.sh
  rm ac_thread.sh
 AC_MSG_RESULT(found)
 ],[
@@ -585,7 +585,7 @@
 your thread library is not available.  Please fill out the ac_thread.sh file.
 ])
   else
-    . ac_thread.sh
+    . ./ac_thread.sh
   fi
 ])
   AC_DEFINE_UNQUOTED(SIGC_PTHREAD_COND_ATTR,$ac_thread_cond_attr_size)
